v2/rscg_examples/SG4MVC/src/MVCDemo/Views/Shared/_Layout.cshtml.css

```
@@ -0,0 +1,48 @@
+﻿/* Please see documentation at https://learn.microsoft.com/aspnet/core/client-side/bundling-and-minification
```

Biome (2.1.2)

[error] Unknown type selector is not allowed.

See MDN web docs for more details.
Consider replacing the unknown type selector with valid one.

(lint/correctness/noUnknownTypeSelector)